wordpress-mobile/WordPress-iOS
# Quick Start: Remove completed list from checklist modal

Part of #18552

## Description
- Removed success screen from QuickStart modal when all tasks are completed.
- Removed the separate completed list inside the modal. Now the task is greyed off in place when completed.

<img src="https://user-images.githubusercontent.com/25306722/169074798-51ba811d-9d17-4c1a-88c0-184f093a1b36.png" width=350> <img src="https://user-images.githubusercontent.com/25306722/169074809-5c552666-61ac-4159-9ae8-413b5fa14ca1.png" width=350>

## Testing Instructions

1. Enable Quick Start
2. Open the Quick Start modal
3. Make sure all tasks are displayed as expected
4. Complete a task
5. Make sure the completed task is greyed out in place
6. Try skipping a task by swiping left
7. Make sure the task is greyed out
8. Complete all tasks
9. Make sure the success screen is no longer displayed.

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

5. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
